src/jit/lower.cpp

```diff
@@ -43,6 +43,7 @@ void Lowering::MakeSrcContained(GenTreePtr parentNode, GenTreePtr childNode)
 {
     assert(!parentNode->OperIsLeaf());
     assert(childNode->canBeContained());
+    childNode->SetAsContained();
 
     int srcCount = childNode->gtLsraInfo.srcCount;
```

Comment:
So we still depend on `gtLsraInfo` being initialized so we can't move containment logic from `TreeNodeInfoInit` to lower?